pkg_version.py

```diff
@@ -0,0 +1,67 @@
+import os

```

Comment:
I'd like this file to be somewhere other than the root of the code base. Since it's just being used by the conda stuff (at least I assume so), maybe it could go in the rkdit-conda repo?

Reply:
I think I have an idea how to make it work. Again, I’ll check it later.

Reply:
I was looking at the load_setup_py_data details in conda build, and it looks like it might accept a path to a recipe directory, where this script could be located (https://github.com/conda/conda-build/blob/master/conda_build/jinja_context.py#L82).